pyproject.toml

```diff
@@ -42,6 +43,10 @@ python = ">=3.8,<3.13"
 # Core project dependencies
 celery = "^5"
 
+# Dependencies specific to Python 3.8
+contextvars = { version = "*", python = "3.8" }
+functools = { version = "*", python = "3.8" }
```

Comment:
Just sanity checking these, [`contextvars`](https://docs.python.org/3.8/library/contextvars.html) and [`functools`](https://docs.python.org/3.8/library/functools.html) are both still provided by the standard library in 3.8.

I'm assuming that these refer to the [contextvars](https://pypi.org/project/contextvars/) and [functools](https://pypi.org/project/functools/) packages on PyPI respectively, both of which are... kinda out of date? Unless it's somehow *absolutely necessary, won't work without 'em*, I **really** don't want to add them here.

Can you confirm (and perhaps explain why) they're a non-negotiable requirement for 3.8 support?

Reply:
good point. i think i added those to my `Pipenv` file 'cause `PyCharm` was complaining that `Package containing module 'contextvars' is not listed in the project requirements` so i added those to be sure.

i think it is better to remove them as i also see them both being available in 3.8 core

Reply:
removed